fiftyone/core/ontology_validation.py

```
@@ -0,0 +1,161 @@
+    errors: list[str] = []
+    for attr in ontology.attributes:
+        if attr.when is None:
+            continue
+        for w in attr.when:
+            try:
+                WhenOperator(w.operator)
+            except ValueError:
+                errors.append(
+                    f"attribute {attr.name!r}: invalid When.operator "
+                    f"{w.operator!r}"
+                )
```

Ruff (0.15.12)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)